pkg/devfile/adapters/common/command.go

```diff
@@ -164,6 +179,35 @@ func validateCommand(data data.DevfileData, command common.DevfileCommand) (err
 	return
 }
 
+// validateCompositeCommand checks that the specified composite command is valid
+func validateCompositeCommand(data data.DevfileData, compositeCommand *common.Composite) error {
+	if compositeCommand.Group.Kind == common.RunCommandGroupType {
+		return fmt.Errorf("compoosite commands cannot belong to the run grouop")
```

Comment:
`composite`. Also I think the message should be `composite commands of run Kind are not supported currently`.

Reply:
👍